src/Views/HardwareView.vala

```diff
@@ -64,7 +64,9 @@ public class About.HardwareView : Gtk.Grid {
         graphics_info.justify = Gtk.Justification.CENTER;
         graphics_info.set_selectable (true);
 
-        var hdd_info = new Gtk.Label (_("%s storage").printf (hdd));
+        ///TRANSLATORS: The first %s represents capacity of system storage.
+        ///The second one is its type: NVMe SSD / eMMC / SATA SSD / HDD.
+        var hdd_info = new Gtk.Label (_("%s %s").printf (hdd, get_storage_type ()));
```

Comment:
Can we use position placeholders here? Otherwise translators won't be able to swap the order if necessary.

Reply:
Fixed. Thank you!